search-service/src/main/kotlin/com/egm/stellio/search/csr/handler/ContextSourceRegistrationHandler.kt

```diff
@@ -0,0 +1,148 @@
+package com.egm.stellio.search.csr.handler
```

Comment:
did not notice before, but handlers are in a `web` package for other business packages. stay consistent.